ui/src/shared/parsing/flux/durations.ts

```diff
@@ -197,7 +197,7 @@ function durationDuration(durationLiteral: DurationLiteral): number {
 function resolveDeclaration(ast: any, name: string): RangeCallPropertyValue {
   const isDeclarator = node => {
     return (
-      get(node, 'type') === 'VariableDeclarator' &&
+      get(node, 'type') === 'VariableAssignment' &&
```

Comment:
Wonder how this changed?

Reply:
it changed in the flux repo as of version 0.8.0